### PR TITLE
Feedbacks photos

### LINKED ISSUE #33: Add Photos of Employees

### PR DESCRIPTION
**Pull request by OlgaUrentseva**
Resolves #33

### CODE REVIEW

File: src/components/organisms/Feedbacks/Content.jsx [hunk 1]
```diff
@@ -1,81 +1,88 @@
...
+const CardsWrapper = styled.div`
+  columns: 25em auto;
+  column-gap: 2em;
 `;
 
-const Feedback = styled.p``;
+const FeedbackCard =
+  typeof window === 'undefined'
+    ? require('./FeedbackCard').default
+    : () => null;
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/Content.jsx:72]: You don't need to do it. Parent `Content` component already using this method.

File: src/components/organisms/Feedbacks/FeedbackCard.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  font-weight: bold;
+`;
+
+const SubTitle = styled.p`
+  margin: 0;
+
+  color: ${({ theme }) => theme.colors.paleText};
+`;
+
+const FeedbackCardWrapper = styled(Card)`
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/FeedbackCard.jsx:50]: No need to use such long name here. Just `Wrapper` should be enough.

File: src/components/organisms/Feedbacks/FeedbackCard.jsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  break-inside: avoid;
+
+  max-width: 100%;
+
+  margin-bottom: 2em;
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/FeedbackCard.jsx:55]: It shouldn't be here. These properties is about layouting. Element itself shouldn't know anything about layout.

File: src/components/organisms/Feedbacks/FeedbackCard.jsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+  page-break-inside: avoid;
+`;
+
+const Feedback = styled.p``;
+
+const FeedbackCard = ({ items, photos }) => {
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/FeedbackCard.jsx:62]: `items` doesn't look like a meaningful name. Here should be list of props:

- `title`
- `teamTime`
- `feedback`
- `photoAlt`
- `photos`

File: src/components/organisms/Feedbacks/FeedbackCard.jsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+          <source
+            srcSet={`${photos.webpSmall} 64w, ${photos.webpBig} 128w`}
+            type="image/webp"
+          />
+          <Image
+            srcSet={`${photos.jpgSmall} 64w, ${photos.jpgBig} 128w`}
+            src={photos.jpgSmall}
+            alt={items.photoAlt}
+            width="568"
+            height="454"
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/FeedbackCard.jsx:80]: It should be
```suggestion
            width="64"
            height="64
```

File: src/components/organisms/Feedbacks/FeedbackCard.jsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import Card, { variants as cardVariants } from '../../atoms/Card';
+
+const Photo = styled.div`
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/FeedbackCard.jsx:4]: It should be `picture` tag with `display: block;`.

All the magic with auto switching between avif/webp/jpg works only with `<picture>`

**Review by DanilAgafonov — CHANGES_REQUESTED** [review]

File: src/components/organisms/Feedbacks/FeedbackCard.jsx [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+  flex: none;
+
+  width: 4em;
+  height: 4em;
+
+  background-color: ${({ theme }) => theme.colors.secondary};
+
+  border-radius: 50%;
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/FeedbackCard.jsx:12]: You can remove it from here. I can see you added `border-radius` to `img` tag

File: src/components/organisms/Feedbacks/FeedbackCard.jsx [hunk 8]
```diff
@@ -1,15 +1,10 @@
 import styled from 'styled-components';
 import Card, { variants as cardVariants } from '../../atoms/Card';
 
-const Photo = styled.div`
-  flex: none;
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/FeedbackCard.jsx]: You need to leave it. Do not remove

File: src/components/organisms/Feedbacks/FeedbackCard.jsx [hunk 9]
```diff
@@ -1,15 +1,10 @@
...
-
+const Picture = styled.picture`
+  display: block;
   width: 4em;
   height: 4em;
-
-  background-color: ${({ theme }) => theme.colors.secondary};
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/FeedbackCard.jsx]: You need to leave it. Do not remove

File: src/components/organisms/Feedbacks/FeedbackCard.jsx [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+const SubTitle = styled.p`
+  margin: 0;
+
+  color: ${({ theme }) => theme.colors.paleText};
+`;
+
+const Wrapper = styled(Card)`
+  break-inside: avoid;
+
+  page-break-inside: avoid;
```
Comment (DanilAgafonov) [inline src/components/organisms/Feedbacks/FeedbackCard.jsx:48]: I think there is no need for these styles here. You have the same thing in `Content` component

File: src/components/organisms/Feedbacks/FeedbackCard.jsx (hunk 9 above)
Comment (DenisZackharov) [reply src/components/organisms/Feedbacks/FeedbackCard.jsx]: @DanilAgafonov  If I leave it, I need to leave border-radius too?

Comment (DanilAgafonov) [reply src/components/organisms/Feedbacks/FeedbackCard.jsx]: See other placed on page where we use images. We usually use `background-color` on `img` tag (not on `picture`). So move it

**Review by DanilAgafonov — APPROVED** [review]

**Pull request merged**